formbricks/formbricks
# Add Loader in Product Delete button

## What does this PR do?
The Product Delete button takes time so this PR adds a loading state to it to keep a track for better UX!

## Type of change

## How should this be tested?
Delete a product from the Product and Settings page and you should be able to see a loader ;)

## Checklist
